docs_src/microservices/device/services/device-mqtt/APIReference.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Device MQTT - API Reference
+---
+
+# Device MQTT - API Reference
+
+Device MQTT inherits its REST APIs from the Device Services SDK.
+
+<swagger-ui src="https://raw.githubusercontent.com/edgexfoundry/device-sdk-go/{{edgexversion}}/openapi/{{api_version}}/device-sdk.yaml"/>
```

Comment:
On the other service we have been referencing the Device Service API Reference page, rather than the swagger directly.